### DISCUSSION TITLE
Whisper Model 405 Error

### DISCUSSION THREAD

**Discussion by KiPSOFT** [deepgram/community#359] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Hello, I'm trying to use the Deepgram Node.js SDK with the whisper model, but I'm getting a 405 error. Here's the initialization code:

const deepgram = client.transcription.live({
    language: "tr",
    smart_format: true,
    model: "whisper-medium",
});

Can you help me with the issue?

### If possible, please attach your code or paste it into the text box.

const deepgram = client.transcription.live({
    language: "tr",
    smart_format: true,
    model: "whisper-medium",
});

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @KiPSOFT, I see you're submitting a live-streaming request to our `whisper-medium` model for Turkish audio. Whisper only supports pre-recorded audio, not streaming, which is the cause of the 405 error. I recommend that you try the base tier of our general model, which is currently the only one that supports Turkish streaming: `"model": "general", "tier": "base"`. 

You can see our [Models Overview](https://developers.deepgram.com/docs/models-overview) docs, listing which models support streaming, and our [Languages Overview](https://developers.deepgram.com/docs/languages-overview) docs to see which models support Turkish.

**Reply by KiPSOFT** [reply]
Thanks

**Comment by KiPSOFT** [comment]
Thanks.
